wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -118,6 +118,19 @@
    */
   public static final String PERSIST_VOLUMES_ATTRIBUTE = "persistVolumes";
 
+  /**
+   * This attribute enables or disables merging plugins while provisioning a workspace. When
+   * enabled, the plugin broker will be configured to attempt to merge plugins where possible (when
+   * two or more plugins share the same image and do not have conflicting settings). Should be
+   * set/read from {@link WorkspaceConfig#getAttributes}.
```

Comment:
maybe we should use Workspace#getAttributes here? @sleshchenko @metlos @sparkoo wdyt?

Reply:
or Devfile#getAttributes

Reply:
Yeah I wasn't 100% on this one, so I opted for what we have for other attributes (e.g. [persistVolumes](https://github.com/eclipse/che/blob/9bf5f6f19e15ad926d28594201b81acc78bcd435/wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java#L108))